### PR TITLE
IGAPP-1368: Fix poi detail design and icon alignment and refactor

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/components/AddressInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  align-self: center;
+  padding: 0 8px;
+`
+
+type AddressInfoProps = {
+  location: LocationModel<number>
+  language: string
+}
+
+const AddressInfo = ({ location, language }: AddressInfoProps): ReactElement => {
```
Comment (steffenkleinle) [inline native/src/components/AddressInfo.tsx:31]: Renamed from PoiDetailItem. Not 100% happy with the name so if you have better ideas let me know :)

File: native/src/components/PoiDetails.tsx [hunk 2]
```diff
@@ -50,10 +45,6 @@ const StyledCategory = styled.Text`
   color: ${props => props.theme.colors.textSecondaryColor};
 `
 
-const ContentWrapper = styled.View`
-  padding-right: 32px;
```
Comment (steffenkleinle) [inline native/src/components/PoiDetails.tsx:54]: @f1sh1918 do you remember why we needed this?

File: native/src/components/AddressInfo.tsx (hunk 1 above)
Comment (LeandraH) [reply native/src/components/AddressInfo.tsx:31]: Only other one I can come up with is AddressBox, and I think AddressInfo is better.

**Review by LeandraH — APPROVED** [review]
I can't reproduce the original issue but I'll believe you that it fixed the issue for your device ;) Also, nice refactoring! Tested on emulated iOS and Android (Nexus 6 API 24 and Pixel 2 API 28)

**Comment by SteffiStuffel** [comment]
I also cannot reproduce the original issue, therefore review from someone else may make more sense.

**Review by sarahsporck — APPROVED** [review]

File: assets/icons/phone.svg
[the remark this answers is not in the document]
Comment (steffenkleinle) [reply assets/icons/phone.svg:1]: Yes. Could you reproduce the issue before my fix?

[the remark this answers is not in the document]
Comment (sarahsporck) [reply assets/icons/phone.svg:1]: Nope. Sadly not...

[the remark this answers is not in the document]
Comment (steffenkleinle) [reply assets/icons/phone.svg:1]: Hmm thats weird...

[the remark this answers is not in the document]
Comment (steffenkleinle) [reply assets/icons/phone.svg:1]: Hmm that is weird but I still think it makes sense to merge this as it didnt make things worse for any of you and better for me.

[the remark this answers is not in the document]
Comment (LeandraH) [reply assets/icons/phone.svg:1]: I agree

**Pull request merged**